docs/index.rst

```diff
@@ -48,11 +48,11 @@ See example::
 
     async def go():
         pool = await aiopg.create_pool(dsn)
-        async with pool.acquire() as cur:
+        async with pool.acquire() as conn:
             async with conn.cursor() as cur:
                 await cur.execute("SELECT 1")
                 ret = []
-                async for row in cur:
+                for row in await cur.fetchall():
```

Comment:
You're right, i didn't catch the difference with this __aiter__ method that is called during async iteration. I think i get it now...